### PR TITLE
Create global.json

### PR DESCRIPTION
**Pull request by ImPosh**

### CODE REVIEW

File: sub-logs/global.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/GlobalwideGames/GlobalwideGames.github.io",
+
+  "owner": {
+    "username": "GlobalwideGames",
+    "email": "joshuavonharten@gmail.com"
+  },
+
+  "target": {
+    "record-type": {
```
Comment (DEV-DIBSTER) [inline sub-logs/global.json:10]: Record type not specified.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by ImPosh — COMMENTED** [review]
fixed

File: sub-logs/global.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  "owner": {
+    "username": "GlobalwideGames",
+    "email": "joshuavonharten@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+       "name": "global", 
+       "value": "https://globalwidegames.github.io"
```
Comment (DEV-DIBSTER) [inline sub-logs/global.json:12]: For a CNAME record, it should be a hostname with no protocols.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by ImPosh — COMMENTED** [review]
fixed it again mb

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**